### DISCUSSION TITLE
From Beta to subscription, howto

### DISCUSSION THREAD

**Discussion by aliasatp** [opengisch/QField#3904] [Q&A]
Hi,
I am plannig to have a organization subscription (I have already done).
Some steps are not so clear for some.
First question:
Do I need also a pro account first or can I manage organization based on a free one?
Second question:
In my organization projects can anyone collaborate (free and pro accounts)?
Thanks

**Comment by suricactus** [comment]
@aliasatp hope my answers help you:

> Do I need also a pro account first or can I manage organization based on a free one?

Organizations are separate accounts, you can manage them with your personal free account. 

> In my organization projects can anyone collaborate (free and pro accounts)?

Yes. The bill will be on the organization based on how many users have been active during the billing period.
